thcrap/src/ntdll.h

```diff
@@ -0,0 +1,59 @@
+/**
```

Comment:
With this file now being present, we could also move the internal Windows structs (which right now are defined in util.h) into this header as well